src/status_im/ui/screens/chat/views.cljs

```diff
@@ -136,25 +135,23 @@
        :no-messages?      no-messages?}]
      [react/text {:style (assoc style/intro-header-description :margin-bottom 32)}
       (str (i18n/label :t/empty-chat-description-one-to-one) contact-name)])
-   (when (and mutual-contact-requests-enabled?
-              (= chat-type constants/one-to-one-chat-type)
-              (or (= contact-request-state constants/contact-request-state-none)
-                  (= contact-request-state constants/contact-request-state-received)
-                  (= contact-request-state constants/contact-request-state-dismissed)))
-     [contact-request])])
+   (when
```

Comment:
should we put this in a helper function which describes what this logic is doing?